pydantic_ai_slim/pydantic_ai/result.py

```diff
@@ -95,7 +95,7 @@ async def _validate_response(
             match = self._output_schema.find_named_tool(message.parts, output_tool_name)
             if match is None:
                 raise exceptions.UnexpectedModelBehavior(  # pragma: no cover
-                    f'Invalid response, unable to find tool: {self._output_schema.tool_names()}'
+                    f'Invalid response, unable to find tool {output_tool_name!r}; expected one of {self._output_schema.tool_names()}'
```

Comment:
this error message and the one below just seemed confusing; I suspect they are hard/impossible to hit in practice but they should still include the relevant information imo